.github/project.yml

```diff
@@ -0,0 +1,4 @@
+release:
+  current-version: 0
```

Comment:
Merging this PR will trigger the release workflow

Reply:
I commented the content of the file